utils/config/examples/daos_server_mdonssd.yml

```diff
@@ -0,0 +1,61 @@
+# Example configuration file for Metadata on SSD.
+
+name: daos_server           # sys group daos_server
+access_points: ['example']  # management service leader (bootstrap)
+# port: 10001               # control listen port, default 10001
+provider: ofi+tcp
+control_log_mask: INFO
+control_log_file: /tmp/daos_server.log
+transport_config:
+  allow_insecure: true
+
+telemetry_port: 9191
+
+engines:
+  -
+    pinned_numa_node: 0
+    targets: 4              # number of I/O service threads per-engine
+    nr_xs_helpers: 1        # count of I/O offload threads per engine
+    fabric_iface: eth0
+    fabric_iface_port: 31316
+    log_file: /tmp/daos_engine.0.log
+
+    env_vars:
+      - FI_SOCKETS_MAX_CONN_RETRY=1
+      - FI_SOCKETS_CONN_TIMEOUT=2000
+
+    # Storage definitions for Metadata on SSD.
+
```

Comment:
A comment to express that the role assignments are optional and will be implicitly assigned based on the configured tiers it is important here. If users are not familiar with how role assignment works then it's best to let them be assigned implicitly.

Reply:
I tried to improve the text in a few places to emphasize that these are optional and normally not needed.